src/api/v1/db/oracledb/ingredients-dao.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash';
+import { openapi } from 'utils/load-openapi';
+import { serializeIngredients } from 'api/v1/serializers/ingredients-serializer';
+import { GetFilterProcessor } from 'utils/process-get-filters';
+import { getConnection } from './connection';
+
+
+const ingredientsGetParameters = openapi.paths['/ingredients'].get.parameters;
+
+const ingredientsColumnNames = {
+  id: 'ID',
+  ingredientType: 'TYPE',
+  name: 'NAME',
+  notes: 'NOTES',
+};
+
+/**
+ * SQL aliases for mapping INGREDIENTS column names to
+ * IngredientResource property names
+ * @const {string}
```

Comment:
You probably want a blank line between description and `@const` as you did in dough: https://github.com/osu-mist/pizza-api/blob/master/src/api/v1/db/oracledb/doughs-dao.js#L11

Same comment with your other JSDoc sections.